docs/docs-beta/docs/integrations/libraries/chroma.md

```diff
@@ -24,43 +24,9 @@ The `dagster-chroma` library allows you to easily interact with Chroma's vector
 pip install dagster dagster-chroma
 ```
 
-### Examples
+### Example
 
-```python
-import os
-import dagster as dg
-from dagster_chroma import ChromaResource, LocalConfig, HttpConfig
-
-@dg.asset
-def my_table(chroma: ChromaResource):
-    with chroma.get_client() as chroma_client:
-        collection = chroma_client.create_collection("fruits")
-
-        collection.add(
-            documents=[
-                "This is a document about oranges",
-                "This is a document about pineapples",
-                "This is a document about strawberries",
-                "This is a document about cucumbers"],
-            ids=["oranges", "pineapples", "strawberries", "cucumbers"],
-        )
-
-        results = collection.query(
-            query_texts=["hawaii"],
-            n_results=1,
-        )
-
-defs = dg.Definitions(
-    assets=[my_table],
-    resources={
-        "chroma": ChromaResource(
-            connection_config=
-                LocalConfig(persistence_path="./chroma") if os.getenv("DEV") else
-                    HttpConfig(host="192.168.0.10", port=8000)
-        ),
-    }
-)
-```
+<CodeExample filePath="integrations/chroma.py" language="python" />
```

Comment:
@dehume could you please rebase on master and update code examples to use `path=` and include the full path from `examples/` (e.g. `docs_beta_snippets/docs_beta_snippets/integrations/chroma.py`?

We've updated the component to be able to access all _examples_.

Reply:
You can probably use this:

```
find docs -type f -print0 | xargs -0 sed -i '' -e 's/filePath="/path="docs_beta_snippets\/docs_beta_snippets/g'
```

https://github.com/dagster-io/dagster/pull/27212